src/modules/site-v2/base/views/tools/phenotype_database.py

```diff
@@ -59,12 +63,89 @@ def results_columns():
 @phenotype_database_bp.route('')
 @cache.memoize(60*60)
 def phenotype_database():
+  """
+    Phenotype Database table (non-bulk)
+  """
+  # Get the list of traits for non-bulk files
+  try:
+    traits_non_bulk = query_phenotype_metadata()
+  except Exception as ex:
+    logger.error(f'Failed to retrieve the list of traits: {ex}')
+    abort(500)
```

Comment:
consider passing a `description=` in the `abort(...)` so that we can use it in the front-end to hint to the end-user what happened.